### PR TITLE
fix: update 2FA code and E2EI enrollment screens UI [WPB-6772]

### PR DESCRIPTION
**Pull request by saleniuk**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6772" title="WPB-6772" target="_blank">BugWPB-6772</a>  Update 2fa validation code screen to follow mocks
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

2FA screen doesn’t look like the other screens, it doesn't match the designs and inset paddings are broken on this screen so the keyboard covers part of the screen and element at the bottom goes under the navigation bar.

### Solutions

- screen title moved to the header bar
- paddings updated to match the designs
- label text updated
- logo at the bottom removed
- insets fixed
- insets also fixed on `E2EIEnrollmentScreen`, it was another screen where it turned out that insets could be done better

#### How to Test

- log in with a user that has 2fa enabled
- navigate through the login process until you are asked to enter the 2fa validation code

### Attachments (Optional)

| Before | After |
| ----------- | ------------ |
|  |  |
|  |  |

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3789](https://10.10.124.134/job/AR-build-pipeline/3789/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(9257434)(9257434)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(9257434)(9257434)-staging-compat.apk)

**Review by Garzas — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/authentication/login/email/LoginEmailVerificationCodeScreen.kt [hunk 1]
```diff
@@ -65,36 +66,26 @@ private fun LoginEmailVerificationCodeContent(
...
-                .constrainAs(logo) {
-                    start.linkTo(parent.start)
-                    end.linkTo(parent.end)
-                    bottom.linkTo(parent.bottom)
-                }
-        )
+    WireScaffold(
+        topBar = {
+            WireCenterAlignedTopAppBar(
+                elevation = 0.dp,
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/authentication/login/email/LoginEmailVerificationCodeScreen.kt]:
```suggestion
                elevation = dimensions().spacing0x,
```

**Review by alexandreferris — APPROVED** [review]
awesome! just have a tiny nitpick

**Review by MohamadJaara — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3810](https://10.10.124.134/job/AR-build-pipeline/3810/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(9264756)(9264756)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(9264756)(9264756)-staging-compat.apk)

**Pull request merged**